akka-http-core/src/main/java/akka/http/javadsl/model/ContentRange.java

```diff
@@ -27,7 +27,7 @@ public static ContentRange create(long first, long last, long instanceLength) {
     }
     @SuppressWarnings("unchecked")
     public static ContentRange create(long first, long last, Option<Long> instanceLength) {
-        return ContentRange$.MODULE$.apply(first, last, ((Option<Object>) (Option) instanceLength).asScala());
+        return ContentRange$.MODULE$.apply(first, last, ((Option<Object>) (Object) instanceLength).asScala());
```

Comment:
it's too bad that we still use $.MODULE$ syntax, we could define an internal accessor class in scaladsl for such things